files/default/neutron-ha-tool.py

```diff
@@ -737,10 +737,21 @@ def target_agent_list(agent_list, agent_type, exclude_agent_host):
     :param exclude_agent_host: hostname of agent to exclude from the list
 
     """
+    agent_info = [
```

Comment:
IMHO is more clear `agent_info` here: shorter name, direct and refers to what contains and not from who. Also the variable is local, and the usage is close.